src/main/drivers/pwm_output.c

```diff
@@ -397,7 +395,9 @@ void pwmWriteDshotCommand(uint8_t index, uint8_t command)
         for (; repeats; repeats--) {
             motor->requestTelemetry = true;
             pwmWriteDshotInt(index, command);
-            pwmCompleteMotorUpdate(0);
+            if (pwmMotorsEnabled) {
```

Comment:
Yes, but this did not interfere with Dshot commands, since the CLI  used `motorControlEnable` in `fc_core.c` to stop motor updates. I just did a test, and Dshot commands are broken if `pwmCompleteDshotMotorUpdate()` is prevented by `!pwmMotorsEnabled` here.